lms/templates/courseware/courseware.html

```diff
@@ -61,7 +61,10 @@
 <%static:css group='style-student-notes'/>
 % endif
 
-<%include file="../discussion/_js_head_dependencies.html" />
```

Comment:
@andy-armstrong Thanks for pointing this out. Made the corresponding change via https://github.com/edx/edx-platform/pull/13483/commits/b8f9caf7c95d9ed293178ddd27f16aabd8a3e6fc.